ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productlist/v1/productlist/price.html

```diff
@@ -0,0 +1,31 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Couldn't we avoid duplicating this HTL script? It's almost similar to the one in `product`, and looks like it's the same as for `searchresults`. We could also have a common template for that, maybe with some extra config to set the extra class for the product component and the "you save" part.

Reply:
The template is now in `core/cif/components/commons/v1/price.html` and is used by all components.